Fix/add handling of empty data and complex layer name

sweco-se1g7y opened the pull request:
Implemented a fix for some situations that created problems in Eskilstuna's environment.

Resolve #65

sweco-se1g7y committed:
fix: added additional checks when receiving SDE data

sweco-se1g7y committed:
fix: added additional checks when receiving SDE data

sweco-se1g7y committed:
feat: cleaned the code

sweco-se1g7y committed:
feat: cleaned the code

sweco-se1g7y committed:
build: updated dependencies

sweco-se1g7y committed:
feat: added support for WMT and WMTS layers

Dalneberg committed:
fix: added exeption-handling

Dalneberg committed:
fix: added exeption-handling

sweco-se1g7y committed:
feat: broke out domains and values to their own kinds

sweco-se1g7y committed:
feat: broke out domains and values to their own kinds

sweco-se1g7y committed:
feat: changed SDE client and Python script to fetch feature classes from all datasets

sweco-se1g7y committed:
feat: changed SDE client and Python script to fetch feature classes from all datasets

sweco-sedalh commented on plugins/arcgis-sde-data-common/src/index.ts:
```diff
@@ -50,7 +50,7 @@ export interface DomainValue {
 export interface ArcGISSDEDomainEntity extends Entity {
   apiVersion: 'geoportia.se/v1alpha1';
-  kind: 'Table';
+  kind: 'ArcGISDomain';
```
As with tables, domains (and domain values) should be more generic, as they also exist in other databases. The `dialect`-pattern can be reused here.

sweco-se1g7y replied:
Fixed!

sweco-se1g7y committed:
feat: made domain kind generic

sweco-se1g7y committed:
feat: made domain kind generic

sweco-se1g7y committed:
feat: added error handling

sweco-se1g7y committed:
fix: added code for breaking dataset, domain and feature class names into namespaces and proper names

sweco-se1g7y committed:
fix: added code for breaking dataset, domain and feature class names into namespaces and proper names

sweco-se1g7y committed:
fix: removed namespaces from Geoserver

sweco-se1g7y committed:
feat: generalised handling of geoserver entities

sweco-se1g7y committed:
fix: added fetch of feature classes without data set

sweco-se1g7y committed:
fix: added fetch of feature classes without data set

sweco-se1g7y committed:
fix: made sure all entered Geoserver entity name follow Backstage requirements

fix: ensured compliant names are used everywhere in the Geoserver data provider

sweco-se1g7y committed:
feat: removed dialect from Geoserver entities

sweco-se1g7y committed:
fix: fixed bug which caused some names to be rejected

sweco-se1g7y committed:
feature: added data sets

sweco-se1g7y committed:
feature: added data sets

sweco-se1g7y committed:
feature: created alternate setup for Eskiltuna's ArcGIS environment

sweco-se1g7y committed:
feature: created alternate setup for Eskiltuna's ArcGIS environment

sweco-se1g7y committed:
doc: added note about Eskilstuna-specifik test script

sweco-se1g7y committed:
doc: added note about Eskilstuna-specifik test script

sweco-se1g7y committed:
feature: ensured all Geoserver name are Backstage compliant

sweco-se1g7y committed:
feature: ensured all Geoserver name are Backstage compliant

sweco-se1g7y committed:
feature: changed Geoserver titles to equal original names

sweco-se1g7y committed:
feature: changed entitty titles to original names

sweco-se1g7y committed:
feature: changed entitty titles to original names

sweco-se1g7y committed:
build: updated dependencies

sweco-se1g7y committed:
build: updated dependencies

sweco-se1g7y committed:
doc: added comments

sweco-se1g7y committed:
doc: added comments

sweco-se1c8x committed:
add and edit readme

sweco-se1c8x committed:
fix the yml file

sweco-se1c8x committed:
change to use crypto instead of CryptoJS

sweco-se1c8x committed:
change to use crypto instead of CryptoJS

sweco-se1c8x committed:
add common plugin and resolve comment

sweco-se1c8x committed:
add common plugin and resolve comment

sweco-se1g7y committed:
build: removed unused dependency

sweco-se1g7y committed:
feat: changed table namespace to always 'default'; removed namespace name from dataset name

sweco-se1g7y committed:
feat: changed table namespace to always 'default'; removed namespace name from dataset name

sweco-se1g7y committed:
build: removed references to CryptoJS

sweco-se1g7y committed:
feat: handled merge problems

sweco-se1g7y committed:
Merge branch 'fix/add-handling-of-empty-data-and-complex-layer-name' of https://github.com/Eskilstuna-kommun/geoportia into fix/add-handling-of-empty-data-and-complex-layer-name
